elasticdl/doc/tutorials/101.md

```diff
@@ -0,0 +1,35 @@
+# 101 Tutorial
+
+101 tutorial contains a series of documents which covers ElasticDL usage under different scenarios: local computer/private cluster/public cloud. It aims to help users get hands on quickly and have a preliminary understanding of ElasticDL.
+
+
+## Overview
+
+
+## ElasticDL on local
+
+Minikube is a tool that makes it easy to run Kubernetes locally. Minikube runs a single-node Kubernetes cluster inside a Virtual Machine (VM) on your laptop for users looking to try out Kubernetes or develop with it day-to-day. Here we use Minikube to run ElasticDL in local laptop.
```

Comment:
Inconsistent uses of "you" and "users". Better to be consistent across the doc

Reply:
Done